src/OSmOSE/utils/formatting_utils.py

```diff
@@ -13,19 +13,23 @@
 
 
 def aplose2raven(
-    df: pd.DataFrame, audio_datetimes: list[pd.Timestamp], audio_durations: list[float]
+    df: pd.DataFrame,
+    audio_datetimes: list[pd.Timestamp],
+    audio_durations: list[float],
 ) -> pd.DataFrame:
-    """
-    Exports an APLOSE formatted result file to Raven formatted DataFrame
-    provided the list of audio files / durations considered for the Raven campaign.
+    r"""Format an APLOSE result DataFrame to a Raven result DataFrame.
+
+    The list of audio files and durations considered for the Raven campaign should be
+    provided to account for the deviations between the advertised and actual
+    file durations.
 
     Parameters
     ----------
     df: Dataframe,
         APLOSE formatted result DataFrame.
 
     audio_datetimes: list[pd.Timestamp]
-        list of non-naïve timestamps from considered audio files.
+        list of tz-aware timestamps from considered audio files.
```

Comment:
Does "non-naive" mean tz-aware? I wasn't sure, and if it doesn't, I don't know what it means 🥸